src/main/java/com/cloudbees/hudson/plugins/folder/properties/FolderCredentialsProvider.java

```diff
@@ -424,88 +476,73 @@ private synchronized boolean updateCredentials(@NonNull Domain domain, @NonNull
             return false;
         }
 
-        @SuppressWarnings({"unchecked", "rawtypes"}) // erasure
-        @Extension(optional = true,ordinal = -1001)
-        // TODO remove credentials 2.0+
-        public static class ActionFactory extends TransientActionFactory<AbstractFolder> {
-            @Override
-            public Class<AbstractFolder> type() {
-                return AbstractFolder.class;
-            }
-            @Override
-            public Collection<? extends Action> createFor(AbstractFolder target) {
-                final FolderCredentialsProperty prop = ((AbstractFolder<?>) target).getProperties().get(FolderCredentialsProperty.class);
-                if (prop != null) {
-                    return Collections.singleton(new CredentialsStoreAction() {
-                        @NonNull
-                        @Override
-                        public CredentialsStore getStore() {
-                            return prop.getStore();
-                        }
-                    });
-                } else {
-                    return Collections.emptySet();
-                }
-            }
-
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        public AbstractFolderProperty<?> reconfigure(StaplerRequest req, JSONObject form) throws FormException {
```

Comment:
To keep the property. If we don't do this then the property will be removed on a form resale as there is no form values in the config.jelly